deployments/dev/images/secondary/environment.yml

```diff
@@ -7,69 +7,69 @@ channels:
 dependencies:
 - python==3.11.*
 - git==2.39.1
+- jupyterhub==5.0.0
```

Comment:
This does not match the version of jupyterhub in `images/hub/Dockerfile`. Our image is using 3.3.7 and according to https://hub.jupyter.org/helm-chart/ that corresponds with jupyterhub 4.1.5. That is also what all of our other images use. (the ones under `deployments/` at least)

Reply:
Thanks Ryan, I will fix that now.